legend-engine-server/src/test/resources/org/finos/legend/engine/server/test/userTestConfig_withFlowProvider.json

```diff
@@ -69,7 +69,10 @@
     "tempPath": "/tmp/",
     "flowProviderClass": "org.finos.legend.engine.authentication.LegendDefaultDatabaseAuthenticationFlowProvider",
     "flowProviderConfiguration": {
-      "_type" : "legendDefault"
+        "_type": "legendDefault",
+        "defaultAWSAccountID": "DEFAULT_AWS_ACCOUNT_ID",
```

Comment:
The config elements should be marked as ignorable. Engine instances that do not use this flow should not be required to provide dummy values.